python/desc/twinkles/sqlite_tools.py

```diff
@@ -29,3 +31,30 @@ def create(self, columns, table, condition=None):
             query += ' ' + condition
         data = [row for row in self.conn.execute(query)]
         return pd.DataFrame(data=data, columns=columns)
+
```

Comment:
Is there a reason to use this rather than use `pd.read_sql_query` or `pd.read_sql_table` ?